integration/spark/spark2/src/test/resources/spark_scripts/spark_alter_table.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Need to deduplicate those files. I think those will only be needed for `app` integration tests

Reply:
I removed all (I think) unnecessary test resources